solc/CommandLineInterface.cpp

```diff
@@ -812,7 +822,7 @@ bool CommandLineInterface::processInput()
 	m_compiler.reset(new CompilerStack(fileReader));
 
 	auto scannerFromSourceName = [&](string const& _sourceName) -> solidity::Scanner const& { return m_compiler->scanner(_sourceName); };
-	SourceReferenceFormatter formatter(cerr, scannerFromSourceName);
+	SourceReferenceFormatter formatter(serr(false), scannerFromSourceName);
```

Comment:
Won't it slip by ``g_hasOutputed`` now, if something is printed using the ``formatter`` below?

Reply:
Yeah, need to fix and add test case

Reply:
Yeah, need to fix and add test case

Reply:
> Won't it slip by `g_hasOutputed` now, if something is printed using the `formatter` below?

Thanks :-)

Reply:
I'm not sure if we should clutter the source reference formatter for this feature. Perhaps it would be better to just set the flag to true in this file here at the places where we use the formatter.